src/module/iotjs_module_udp.cpp

```diff
@@ -307,52 +282,117 @@ JHANDLER_FUNCTION(Close) {
 }
 
 
-JHANDLER_FUNCTION(GetSockeName) {
-  IOTJS_ASSERT(!"Not implemented");
+GetSockNameFunction(UdpWrap, udp_handle, uv_udp_getsockname);
 
-  iotjs_jhandler_return_null(jhandler);
+JHANDLER_FUNCTION(GetSockeName) {
+  DoGetSockName(jhandler);
 }
 
+#define X(fn)                                                                 \
```

Comment:
First time i dind't name it because its role is very simple to set options.  If you prefer longer name, then how about `IOTJS_UV_SET_SOCKOPT` ?